web/src/components/dashboard/anytime-actions/AnytimeActionDropdown.tsx

```diff
@@ -40,12 +40,10 @@ export const AnytimeActionDropdown = (props: Props): ReactElement => {
   };
 
   const licenseReinstatementMatch = (action: AnytimeActionLicenseReinstatement): boolean => {
-    return !!(
-      action.industryIds &&
-      industryId &&
-      action.industryIds.includes(industryId) &&
-      business.licenseData?.status === "EXPIRED"
-    );
+    const licenseNameFromAnytimeAction = action.licenseName;
+    const licenseStatus = business?.licenseData?.licenses?.[licenseNameFromAnytimeAction]?.licenseStatus;
+
+    return licenseStatus === "EXPIRED";
```

Comment:
Do we need to restrict what anytime actions appear, i.e. only show the ones relevant to industry

Reply:
Spoke to the team; no change is required.